### PR TITLE
EVG-3524: display container pool mapping information in UI

### PR DESCRIPTION
**Pull request by li-cindy**
looks like this when the ID in "Pool ID" exists:
screen shot 2018-07-10 at 1 38 32 pm

### CODE REVIEW

File: public/static/js/distros.js [hunk 1]
```diff
@@ -1,8 +1,9 @@
-mciModule.controller('DistrosCtrl', function($scope, $window, $location, $anchorScroll, mciDistroRestService) {
+mciModule.controller('DistrosCtrl', function($scope, $window, $location, $anchorScroll, $filter, mciDistroRestService) {
 
   $scope.readOnly = !$window.isSuperUser;
 
   $scope.distros = $window.distros;
+  $scope.containerPools = $window.containerPools;
```
Comment (john-m-liu) [inline public/static/js/distros.js]: You don't need to set this if you don't need this in the HTML

Comment (li-cindy) [reply public/static/js/distros.js]: But don't I need to set it since I am using it in this JS file?

Comment (john-m-liu) [reply public/static/js/distros.js]: You can access $window.containerPools in the js

**Comment by john-m-liu** [comment]
Sorry for not noticing this earlier, but the UI element to choose the pool should really be a dropdown list. Use a select element for it where the choices are bound to the list in $window.containerPoolIds

**Comment by li-cindy** [comment]
@john-m-liu Choosing a pool is now done with a dropdown list.

**Pull request merged**